src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,129 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+import { ItemsPerPage } from '../../types/ItemsPerPage';
+
+type Item = string;
+
+const getItemsPerPage = (
+  items: Item[],
+  countPerPage = 5,
+): ItemsPerPage<Item> => {
+  const itemsPerPage: ItemsPerPage<Item> = {};
+
+  items.forEach((item, index) => {
+    const currentPage = String(Math.floor(index / countPerPage) + 1);
+
+    if (itemsPerPage[currentPage]) {
+      itemsPerPage[currentPage].push(item);
+    } else {
+      itemsPerPage[currentPage] = [item];
+    }
+  });
+
+  return itemsPerPage;
+};
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}: Props) => {
+  const items: Item[] = getNumbers(1, total).map((n) => `Item ${n}`);
```

Comment:
`Item ${n}` is basically the content, I think we should render it in JSX when you map through the all possible numbers